LLAP/llap.sh

```diff
@@ -0,0 +1,438 @@
+#!/bin/bash
+
+#    Copyright 2021 Google, Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+set -euxo pipefail
+
+##install xml modifiction tool....
+
+apt-get install -y xmlstarlet
```

Comment:
We already have bdconfig to get/set values from xml file, used in most init actions. Example, 
set_property: https://github.com/GoogleCloudDataproc/initialization-actions/blob/master/hbase/hbase.sh#L58
get_property_value: https://github.com/GoogleCloudDataproc/initialization-actions/blob/master/presto/presto.sh#L131

Reply:
so should I make the change? xmlstarlet works pretty well. If I had known about this before I'd have used the BDConfig

Reply:
Yes please change. we already have a util, so no need to install a separate one.

Reply:
fixed